### ISSUE TITLE
Is there any way to reuse same session with multicookie Bard?

### ISSUE THREAD

**Issue by SuSl1KK** [dsdanielpark/Bard-API#258]
I've seen the reusable session documentary in the README, but looks like it is created for single-cookie. Im using browser_cookie3 to update PSIDCC and PSIDTS cookies everytime they get changed, but it also creates new conversation as result.

**Comment by dsdanielpark** [comment]
Currently, as this API is not an official service, it is reported that cookie values are refreshed every 15-20 minutes (previously, there were longer durations without a rate limit, but now it exists).

Therefore, we cannot address the issue of new conversations being initiated after cookie values are refreshed. To maintain a conversation or its context, we recommend either coding like GPT to store conversation content in a database and then throw some of it into the mix to maintain context (by appending some conversation content to a list or attaching a database), or finding an alternative workaround.

Some codes for maintaining conversation IDs or context were implemented, but they weren't very effective.

In conclusion, it is very difficult to change or set the hyperparameters of the generative model through this API, and we cannot assist at the package level, so I am closing this issue.

However, as others might be able to help with this matter, this issue will remain open for 7 days before closing. Thank you.

Please feel free to open a pull request if there are any new features or solutions. Wishing you a prosperous new year and a great day.
